test/python/taskbasedsimulator/benchmark.py

```diff
@@ -1,148 +1,422 @@
+import csv
 import datetime
+import math
+import os.path
 import time
+
+import numpy
+import cmath
+
 from mqt.ddsim.pathqasmsimulator import PathQasmSimulator, get_simulation_path
 from mqt import ddsim
-from qiskit import transpile
+from qiskit import *
+from mqt.bench import get_one_benchmark
+from pathlib import Path
 
-from test.python.generate_benchmarks import *
+# from test.python.generate_benchmarks import *
 from typing import Union
 
 
 def execute_circuit(qc: QuantumCircuit, backend, shots: int, mode: Union[str, ddsim.PathSimulatorMode] = 'sequential', **options):
```

Comment:
this function is never used. consider removing it.

Reply:
done and will be included in the latest commit